store/servers.ts

```diff
@@ -0,0 +1,99 @@
+import { ActionTree, MutationTree } from 'vuex';
+
+export interface ServerState {
+  Id: string;
+  LocalAddress: string;
+  ServerAddress: string;
+  OperatingSystem: string;
+  ProductName: string;
+  ServerName: string;
+  StartupWizardCompleted: boolean;
+  Version: string;
+}
```

Comment:
Just a thought, but I think something like this would be even better, and could likely allow for easier multi-server support down the line:

```typescript
export interface ServerInfo {
    address: string;
    publicInfo: PublicSystemInfo;
}

interface ServersState {
    [key: string]: ServerInfo;
}
```

The idea being that we'd have an array of servers, with the id as key and the result of the public info endpoint as one of the children. We also get the actual address entered, to ensure we have it somewhere.

Then we can gave a getter to filter by ID (for example), that gets us only one result (in a different form, if we wish).

Reply:
Cool. I've added this. I think I've done it in the right way but please correct me if I'm wrong